src/orbit-chain-governance/factories/ParentChainGovFactory.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../../L1ArbitrumTimelock.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "@arbitrum/nitro-contracts/src/bridge/IInbox.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+/// @title Factory that deploys governance chain contracts for cross chain governance
+/// @notice Requires core nitro contracts (i.e., inbox), and Upgrade Executor, and a Proxy Admin to
+/// be deployed on the parent chain, and a governance timelock be deployed on the governance chain
+/// via GovernanceChainGovFactory.
+contract ParentChainGovFactory is Ownable {
+    bool private done = false;
+
+    event Deployed(L1ArbitrumTimelock timelock);
```

Comment:
Maybe add inbox address here. There will be many factories on parent chain, so that's a good way to track deployments